packages/scaffold-ui/package.json

```diff
@@ -47,6 +47,7 @@
     "url": "https://github.com/reown-com/appkit/issues"
   },
   "devDependencies": {
+    "@open-wc/testing": "4.0.0",
```

Comment:
aren't you missing `vitest`?

Reply:
i think we already have `vitest`. I can see it in `package.json`